packages/x-data-grid/src/components/panel/filterPanel/GridFilterInputSingleSelect.tsx

```diff
@@ -76,9 +78,10 @@ function GridFilterInputSingleSelect(props: GridFilterInputSingleSelectProps) {
     placeholder,
     tabIndex,
     label: labelProp,
+    variant = 'standard',
     isFilterActive,
     clearButton,
-    InputLabelProps,
+    inputLabelProps,
```

Comment:
Why is this change necessary?

Also I'm not sure about the naming convention, but I think I might have seen other cases of `UpperCaseProps`, if so then this one should stay the same for consistency.

Reply:
Sorry I missed that part you were right. Now it is fixed.